chronicles/options.nim

```diff
@@ -216,6 +216,12 @@ proc syntaxCheckStreamExpr*(n: NimNode) =
       error &"Invalid stream definition. " &
              "Please use a bracket expressions such as 'stream_name[sinks_list]'."
 
+proc isLogFormatUsed*(conf: Configuration, format: LogFormat): bool =
```

Comment:
It would be OK if this function refers internally to the global `config` variable, so the first parameter can be dropped.